scripts/swagger-formatting.js

```diff
@@ -0,0 +1,144 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { Command } = require('commander');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('fs');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path');
+
+const getSwaggerMethodName = (methodName) => {
+  return 'Api' + methodName.slice(0, 1).toUpperCase() + methodName.slice(1);
+};
+
+const getSwaggerFormat = (methodName) => {
+  return `export const ${methodName} = (summary: string) => {
+  return applyDecorators(
+    ApiOperation({ summary }),
+    ApiSuccessResponse(HttpStatus.OK, {}),
+    ApiFailureResponse(HttpStatus.BAD_REQUEST, []),
+  );
+};
+`;
+};
+
+const getBaseImport = () => {
+  return `import { applyDecorators, HttpStatus } from '@nestjs/common';
+import { ApiOperation } from '@nestjs/swagger';
+import { ApiFailureResponse } from '@src/decorators/api-failure-response.decorator';
+import { ApiSuccessResponse } from '@src/decorators/api-success-response.decorator';
+`;
+};
+
+const program = new Command();
+
+program
+  .description(
+    `컨트롤러 기준으로 controller.swagger.ts 를 생성해줍니다.
+-m 은 필수 옵션이고 -c 를 주지 않는다면 module name 기준으로 컨트롤러를 찾습니다.
+ex) npm run swagger -m "member" => member.controller.ts 를 찾아 member.swagger.ts 를 생성합니다.
+ex) npm run swagger -m "member" -c "member-like" => member module 안에있는 member-like.controller.ts 를 찾아 member-like.swagger.ts 를 생성합니다.`,
+  )
+  .requiredOption('-m, --module <module>', 'module name')
+  .option('-c, --controller <controller>', 'controller name')
+  .parse();
+
+const options = program.opts();
+const moduleName = options.module;
+const controllerName = options.controller || moduleName;
+const controllerFileName = controllerName + '.controller.ts';
+
+let dir = __dirname.split('/');
+dir.pop();
+dir = path.join(dir.join('/'), 'src', 'modules', options.module, 'controllers');
```

Comment:
```suggestion
let dir = path.join('src', 'modules', options.module, 'controllers');
```

__dirname로 뽑아오는 path (C:\Users\subro\Desktop\the-pool-api\scripts)중 / 가
존재하지 않는데 splic을 하신 이유가 있을까요?

Reply:
__dirname 값이 /Users/rokwon/private/github/the-pool-api/scripts 이런식으로 나오지 않나요? 만약 맥과 윈도우 결과값이 다르다면 ㄴㅇㄱ

Reply:
윈도우는 생각 못해봤네요 윈도우환경에서 확인해보고 코멘트 남길게요

Reply:
https://p-iknow.netlify.app/node-js/path-moudle/

유닉스 기반의 운영체제와 달리 Windows는 C:\Users\subro 처럼 "\" 를 사용해 폴더를 구분한다고 하네요!